user/tests.py

```diff
@@ -147,3 +147,56 @@ def test_post_blank_password(self):
     self.assertTemplateUsed(self.response, 'user/signup.html')
 
 
+class LoginSuccessTest(TestCase):
+  def setUp(self):
+    Account.objects.create_user(username='people', email='test@test.test', password='testpassword', age='1')
+    self.home_url = reverse('home')
+    self.login_url = reverse('login')
+
+  def test_login_success(self):
+    self.login_data = {'username':'people', 'password':'testpassword'}
+    self.login_response = self.client.post(self.login_url, self.login_data)
```

Comment:
```suggestion
    login_data = {'username':'people', 'password':'testpassword'}
    login_response = self.client.post(self.login_url, self.login_data)
```

setUpで定義する変数以外は`self.`の形で記述しなくていいと思います！